doc/man7/OSSL_PROVIDER-FIPS.pod

```diff
@@ -335,6 +335,8 @@ The FIPS module passes the following descriptions(s) to OSSL_SELF_TEST_onbegin()
 
 =item "RSA" (B<OSSL_SELF_TEST_DESC_PCT_RSA_PKCS1>)
 
+=item "RSA" (B<OSSL_SELF_TEST_DESC_PCT_RSA>)
+
```

Comment:
Both are given the name "RSA"...  isn't that going to be confusing?

Reply:
The main reason for changing it was because the actual macro name included the padding type which is no longer accurate. The PCT itself is still an RSA PCT.